src/segyviewlib/segywidgetcollection.py

```diff
@@ -20,45 +20,104 @@ def blocked_update(ctxt, current_ctxt):
 
 
 class SegyTabWidget(QWidget):
-    def __init__(self, segywidgets, parent=None):
+    def __init__(self, segywidgets=[], parent=None):
```

Comment:
It is generally considered bad form to use mutable objects as default values; in Python everything is a reference and the reference used in the method will be shared with calling scope. It is better to use `None`:

```python
def __init__(self, segywidgets = None, ...):
    ...
    ...
    if segywidgets is None:
        self._context = None
    else:
        self.initialize( )
```